MVC/controller.js

```diff
@@ -1,7 +1,10 @@
-const {fetch} = require("./models")
+const {fetchTopics} = require("./models")
 
-exports. get = (req, res, next) => {
+exports. getTopics = (req, res, next) => {
     // console.log('controller.js bounjour');
-    fetch()
+    fetchTopics()
+    .then((topics) => {
+        res.status(200).send(topics)
```

Comment:
We should send back data to the client on a meaningful key e.g, they should be able to access it on body.topics - at the moment we are sending an object through but not on a key of topics - how could we ensure the user can access the topics array on a key of topics? :)